### DISCUSSION TITLE
flux supported?

### DISCUSSION THREAD

**Discussion by fahadshery** [YanWenKun/ComfyUI-Docker#58] [General]
Hi,

I am currently using https://github.com/AbdBarho/stable-diffusion-webui-docker

That doesn't have flux support because I get the error of not being able to load the model.

Have you tested flux?

Cheers

**Comment by YanWenKun** [comment]
Of course! 😄

I would suggest using [cu124-megapak](https://github.com/YanWenKun/ComfyUI-Docker/blob/main/cu124-megapak/README.adoc) image.

Also:

* Try [Comfy's example workflow](https://comfyanonymous.github.io/ComfyUI_examples/flux/)

* Or an [all-fp8 workflow](https://github.com/user-attachments/files/16728158/flux_schnell_fp8.json)

* If you're using 40 series GPU, try add `--fast` to CLI_ARGS, see: #57

**Comment by fahadshery** [comment]
thanks very much...

do you have a `docker-compose.yml`?

**Reply by fahadshery** [reply]
sorry found them in the project folders...
